# Parameters for slam_mapper_node_updated.py
map_resolution: 0.05
map_width_meters: 20.0
map_height_meters: 20.0
map_publish_rate: 0.5
map_frame_id: "map"       # Should match optimizer output
odom_frame_id: "odom"      # Should match robot_localization output
base_frame_id: "ente/base" # Your robot's base frame
line_segment_length: 0.3
loop_closure_dist_thresh: 0.5 # For logging potential loops
loop_closure_min_keyframes: 15
keyframe_dist_thresh: 0.3
keyframe_angle_thresh: 0.1745 # 10 degrees in radians
tf_lookup_timeout: 0.1
